packages/metrics/src/features/locationStatistics/service.ts

```diff
@@ -109,16 +108,12 @@ export async function getLocationStatistics(
   )
   if (locationId) {
     const location = await fetchLocation(locationId, authHeader)
-    if (getLocationType(location) !== 'CRVS_OFFICE') {
-      return getAdminLocationStatistics(
-        location,
-        registrarsCountByLocation,
-        populationYear,
-        authHeader
-      )
-    } else {
-      return getCRVSOfficeStatistics(location, registrarsCountByLocation)
```

Comment:
Remove crvs office related function as we do not fetch when location is crvs office